pkg/client/client.go

```diff
@@ -90,10 +90,11 @@ type CacheOptions struct {
 type NewClientFunc func(config *rest.Config, options Options) (Client, error)
 
 // New returns a new Client using the provided config and Options.
-// The returned client reads *and* writes directly from the server
-// (it doesn't use object caches).  It understands how to work with
-// normal types (both custom resources and aggregated/built-in resources),
-// as well as unstructured types.
+// The returned client reads from a local cache or directly from the API server,
```

Comment:
Is this `or` intentional?  Did the previous client not only read directly from the apiserver?

Reply:
Never mind, I see that if you set `options.Cache` it will read from the cache and not directly from the api server.  However the default case would return a client that reads directly from the api-server.

Reply:
Hi @troy0820, thanks for the comments. The objective is to disambiguate the comments and to reflect that the read could be performed on a local cache (and not necessarily on the API server).

Reply:
Could you explain in the comment in which cases a cached client is returned? (By referring to the options passed to this function)

Reply:
@inteon Thanks for pointing this out. The comment is updated accordingly. Could you please take another look ?